packages/pdf.lua

```diff
@@ -89,15 +89,33 @@ SILE.registerCommand("pdf:link", function (options, content)
   })
 end)
 
+SILE.registerCommand("pdf:metadata", function (options, content)
+  local key = SU.required(options, "key", "pdf:metadata")
+  local val = SU.required(options, "val", "pdf:metadata")
+  SILE.typesetter:pushHbox({
+    value = nil,
+    height = 0,
+    width = 0,
+    depth = 0,
+    outputYourself = function (self,typesetter)
+      SILE.outputter._init()
+      pdf.metadata(key, val)
+    end
+  })
+end)
+
 return { documentation = [[\begin{document}
 The \code{pdf} package enables (basic) support for PDF links and table-of-contents
-entries. It provides the three commands \command{\\pdf:destination}, \command{\\pdf:link}
-and \command{\\pdf:bookmark}.
+entries. It provides the four commands \command{\\pdf:destination}, \command{\\pdf:link},
+\command{\\pdf:bookmark}, and \command{\\pdf:metadata}.
```

Comment:
Thank you for the Oxford comma here, the _only_ sane way to use commas in English. :wink: